packages/preview/tauthesis/0.3.2/typst.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "tauthesis"
```

Comment:
You'll need to change the name since template names must have at least one unique, creative component so that multiple people can create a template for the same instutition without one having an unfair advantage. For organization-specific templates, we recommend the naming scheme <adjective>-tau-thesis. See the [submission guidelines](https://github.com/typst/packages?tab=readme-ov-file#submission-guidelines) for details. You can also take a look [at the existing templates](https://typst.app/universe/search?kind=templates) for inspiration.

Reply:
May I leave the hyphen out from between `tau` and `thesis` and simply come up with a unique-ish prefix?

Reply:
The package has been renamed as `scholarly-tauthesis`.